source/includes/administration/_service_providers.md

```diff
@@ -304,3 +304,57 @@ curl "https://cloudmc_endpoint/rest/service_providers/f9dea588-d7ab-4f42-b6e6-4b
 ```
 
 Delete a specific service provider. 
+
+<!-------------------- GET SERVICE PROVIDER METADATA -------------------->
+
+
+#### Get service provider metadata file
+
+`GET /service_providers/:id`
```

Comment:
isn't it `GET /service_providers/:id/metadata` ?

Reply:
:0 good eye
sure is

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/184/commits/60494063e44a79425684c7894d75e589e93a338f